recipes/sspace/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/
+mv * $PREFIX/bin
```

Comment:
I think it would be better to use $PREFIX/opt/sspace and link it. You could look at the krona recipe. Otherwise you are copying several folders and pdfs to the bin folder.